src/libraries/System.Text.Json/src/System/Text/Json/AppContextSwitchHelper.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Text.Json
+{
+    internal static class AppContextSwitchHelper
+    {
+        private const string SourceGenReflectionFallbackEnabled_SwitchName = "System.Text.Json.Serialization.EnableSourceGenReflectionFallback";
```

Comment:
My gut says all of our switches start with "Switch.".  But maybe I'm misremembering?

```suggestion
        private const string SourceGenReflectionFallbackEnabled_SwitchName = "Switch.System.Text.Json.Serialization.EnableSourceGenReflectionFallback";
```

Reply:
Certainly all the ones I see at https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/retargeting/4.7.2-4.8 start with "Switch."

Reply:
IMO - that's old style that we have been moving away from. All the new ones I've seen haven't started with `Switch.`:

https://github.com/dotnet/sdk/blob/990a1a0809e3a843263bdb8d33e34d3801d44805/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L419-L528

ex:

* System.GC.Concurrent
* System.Globalization.Invariant
* System.Runtime.TieredCompilation


https://github.com/dotnet/runtime/blob/ad8debf62d9ae538c4e1a2f473a5a86377eabbc8/src/libraries/System.Private.CoreLib/src/Microsoft/Win32/SafeHandles/SafeFileHandle.Unix.cs#L38